### DISCUSSION TITLE
Article about maestral in Genbeta (Spanish)

### DISCUSSION THREAD

**Discussion by erseco** [samschott/maestral#483] [General]
A Spanish blog published an article about maestral

https://www.genbeta.com/herramientas/iba-a-abandonar-dropbox-cuando-descubri-maestral-este-cliente-gratuito-le-devuelve-brillo-perdido-elimina-sus-restricciones

**Comment by samschott** [comment]
Oh, nice! I'll need to run it through Google translator to understand it fully but I think I get the gist of it from the screenshots 😄
